cubeviz/controls/flux_units.py

```diff
@@ -327,6 +333,22 @@ def set_data(self, data):
         if wcs is not None:
             self.wcs = wcs
 
+
+    def add_data(self, data):
```

Comment:
It looks like this method isn't used anywhere. Is it necessary?

Reply:
twas unnecessary, removed.